packages/frontend/src/components/AccountVerification/AccountVerification.styled.ts

```diff
@@ -9,8 +9,16 @@ import { Breakpoint } from "src/enums/breakpoint.enum";
 
 export const PageWrapper = styled(FlexContainer)`
   height: 100vh;
-`;
+  margin: 0 auto;
+  padding: 0 40px;
 
+  @media ${Breakpoint.Desktop} {
+    max-width: 1280px;
```

Comment:
takie wartości do enuma bo zakładam że to będzie pewnie szerokośc grida dla desktopu